functionaltests/src/test/java/specs/BaseFixture.java

```diff
@@ -165,28 +185,100 @@ protected String createCartAndOrderForPayment(final Payment payment, final Strin
         return orderNumber;
     }
 
-    protected String getUnconfirmedVisaPseudoCardPan() {
-        return getConfigurationParameter(TEST_DATA_VISA_CREDIT_CARD_NO_3_DS);
+    private static String PSEUDO_CARD_PAN;
+
+    /**
+     * Fetch new or get cached pseudocardpan from Payone service based on supplied test VISA card number
+     * (see {@link #getTestDataVisaCreditCardNo3Ds()}).
+     * Since Concordion is multi-threading - use synchronized lazy initialization.
+     */
+    synchronized static protected String getUnconfirmedVisaPseudoCardPan()  {
+
+      //curl --data "request=3dscheck&mid=$PAYONE_MERCHANT_ID&aid=$PAYONE_SUBACC_ID&portalid=$PAYONE_PORTAL_ID&key=$(md5 -qs $PAYONE_KEY)&mode=test&api_version=3.9&amount=2&currency=EUR&clearingtype=cc&exiturl=http://www.example.com&storecarddata=yes&cardexpiredate=2512&cardcvc2=123&cardtype=V&cardpan=<VISA_CREDIT_CARD_3DS_NUMBER>"
+
+      if (PSEUDO_CARD_PAN == null) {
+        String cardPanResponse = null;
+        try {
+          cardPanResponse = Unirest.post("https://api.pay1.de/post-gateway/")
+              .fields(ImmutableMap.<String, Object>builder()
+                  .put("request", "3dscheck")
+                  .put("mid", getTestDataPayoneMerchantId())
+                  .put("aid", getTestDataPayoneSubaccId())
+                  .put("portalid", getTestDataPayonePortalId())
+                  .put("key", Hashing.md5().hashString(getTestDataPayoneKey(), Charsets.UTF_8).toString())
+                  .put("mode", "test")
+                  .put("api_version", "3.9")
+                  .put("amount", "2")
+                  .put("currency", "EUR")
+                  .put("clearingtype", "cc")
+                  .put("exiturl", "http://www.example.com")
+                  .put("storecarddata", "yes")
+                  .put("cardexpiredate", "2512")
+                  .put("cardcvc2", "123")
+                  .put("cardtype", "V")
+                  .put("cardpan", getTestDataVisaCreditCardNo3Ds())
+                  .build())
+              .asString().getBody();
+        } catch (Throwable e) {
+          LOG.error("Exception in fetching pseudocardpan: ", e);
+          throw new RuntimeException("Exception in fetching pseudocardpan", e);
+        }
+
+        Pattern p = Pattern.compile("^.*pseudocardpan\\s*=\\s*(\\d+).*$", CASE_INSENSITIVE | DOTALL);
+        Matcher m = p.matcher(cardPanResponse);
+
+        if (m.matches()) {
+            assert PSEUDO_CARD_PAN == null : "PSEUDO_CARD_PAN multiple initialization";
+            PSEUDO_CARD_PAN = m.group(1);
+            LOG.info("Fetched pseudocardpan {}", PSEUDO_CARD_PAN);
```

Comment:
It makes tests outputs analyzing easier: i can check that payments were created with correct pseudocardpan.